src/main/java/com/SurveyMonkey/controller/DropDownQuestionController.java

```diff
@@ -0,0 +1,47 @@
+package com.SurveyMonkey.controller;
+
+import com.SurveyMonkey.model.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class DropDownQuestionController {
+    List<DropDownQuestion> dropDownQuestions = new ArrayList<>();
+    @Autowired
+    private CreateSurveyRepository createSurveyRepository;
+    @Autowired
+    private DropDownRepository dropDownRepository;
+
+
+    @PostMapping("/saveDropDownQuestion")
+    public ResponseEntity<Object> addQuestion(@RequestBody DropDownQuestion dropDown) {
+        dropDownQuestions.add(dropDown);
```

Comment:
This should be:
 qtype.add(dropDown);

Reply:
Made the required changes it is now QType